packages/driver/src/cy/commands/screenshot.ts

```diff
@@ -98,6 +98,10 @@ const scrollOverrides = (win, doc) => {
   // hide scrollbars
   doc.documentElement.style.overflow = 'hidden'
 
+  // this body class is used to set some other overflow-related CSS
+  // around the resizable panels in the Runner
+  document.querySelector('body')?.classList.add('screenshot-scrolling')
```

Comment:
Not sure why TS is happy with this, if `document.querySelector('body')?` is ever null (which it shouldn't be) it would be safer to use `document.querySelector('body')?.classList?.add('screenshot-scrolling')`

I would just do `document.body.classList.add('screenshot-scrolling')`

Reply:
I got warnings in vscode when I did plain `document.body` so not sure what's up.